### PR TITLE
Add Integration Tests to CI

### PR DESCRIPTION
**Pull request by dzou**
Add integration tests to CI by supplying encrypted service account key.

status: pending - we'll need to determine what set of integration tests is appropriate to be run. We shouldn't necessarily run all integration tests b/c certain ones might take very long time.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +16,19 @@ addons:
 install:
   - ./mvnw -T 1.5C clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q
 
+before_install:
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+    openssl aes-256-cbc -K $encrypted_5bb5dbfd81fd_key -iv $encrypted_5bb5dbfd81fd_iv -in travis.tar.gz.enc -out travis.tar.gz -d
+    tar -xzf travis.tar.gz;
+    export GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/travis/admin.json;
+    export GOOGLE_CLOUD_PROJECT=cloud-spanner-hibernate-ci;
+    export INTEGRATION_TEST_FLAGS="-Dit.hibernate=true"
```
Comment (meltsufin) [inline .travis.yml]: Missing semicolon?

Comment (dzou) [reply .travis.yml]: Nice catch. Done.

File: .travis.yml [hunk 2]
```diff
@@ -16,9 +16,19 @@ addons:
 install:
   - ./mvnw -T 1.5C clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q
 
+before_install:
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+    openssl aes-256-cbc -K $encrypted_5bb5dbfd81fd_key -iv $encrypted_5bb5dbfd81fd_iv -in travis.tar.gz.enc -out travis.tar.gz -d
```
Comment (meltsufin) [inline .travis.yml]: semicolon here too?

Comment (dzou) [reply .travis.yml]: Done.

Comment (meltsufin) [reply .travis.yml]: Don't see it. I think you modified the wrong line.

Comment (dzou) [reply .travis.yml]: Should be good now. 

Now the build/test runs but getting error related to sonar plugin: https://travis-ci.org/GoogleCloudPlatform/google-cloud-spanner-hibernate/builds/505409847?utm_source=github_status&utm_medium=notification

Any ideas? I haven't yet added `&& [ "$TRAVIS_PULL_REQUEST" == "false" ]` to the before_install yet; might be related to that?

Comment (meltsufin) [reply .travis.yml]: Yeah, the sonar build on master has to succeed first and I'm not sure why it has been failing. The setup seems the same as for S-C-G. Let's just merge for now.

**Comment by dzou** [comment]
PTAL, I merged Chengyuan's changes fixing the tests; now the CI is passing.

**Pull request merged**